README.md

```diff
@@ -87,13 +87,35 @@ select show_cat('Mr. Sleepy', false);
 
 ## Installation
 
-Install https://gitlab.com/jobol/mustach, then on this repo:
+Clone the repo and submodules:
 
 ```
-$ make && make install
+git clone --recurse-submodules https://github.com/PostgREST/plmustache
 ```
 
-Tested on Postgres 12, 13, 14, 15, 16.
+Build mustach:
+
+```
+cd mustach
+make && sudo make install
+sudo ldconfig
```

Comment:
This is required on `Pop!_OS 22.04 LTS`, likely ubuntu/debian as wel. But it wasn't required when building with Nix.

Maybe there's a way to avoid it...

Reply:
Note: I got the `ldconfig` hint from https://github.com/laurenz/oracle_fdw/issues/440#issuecomment-737054845